sections/method/preprocessing.tex

```diff
@@ -1,51 +1,146 @@
 \section{Preprocessing}\label{sec:prepro}
 
-Our preprocessing consists of several steps, each of which will be described in more detail.
-The steps are:
-\vejleder[inline]{give examples of each of the preprocessing steps}
-\begin{enumerate}
+In this section we will give a detailed overview over our method of preprocessing documents.
+Our preprocessing steps were inspired by \cite{quanti}, who give do a thorough  preprocessing before applying LDA. 
+Our preprocessing consists of several steps, each of which will be described in more detail, along with a running example over part of a real document.
+\begin{enumerate}[label=\alph*]
 	\item Article Extraction
 	\item Initial Document Filtering
-	\item Stopwords and Word Frequency
+	\item Stopword and Word Frequency Filtering
 	\item Word Databases / POS Tagging
 	\item Stemming
-	\item Repeat step 2 and Stopwords
 	\item Construct Files
 \end{enumerate}
 
+\begin{table}
+	\begin{tabular}{c|c|c}
+		Preprocessing Step & Documents & Words\\
+		Article Extraction & 63266 & unknown \\ 
+		Initial Document Filtering & 32201 & 190133 \\ 
+		Stopword and Word Frequency Filtering & 32201 & 16406 \\
+		Word Databases / POS Tagging & 32201 & 14537 \\
+		Stemming / Lemmatization & 32201 & 8691 / 9934\\
+	\end{tabular}
+	\label{tab:prepro_doc_word}
+	\caption{Amount of documents and unique words remaining after each preprocessing step.}
+\end{table}
+
 \paragraph{Article Extraction}
-Documents are loaded from \texttt{xml} files, and their ID, title, and body contents extracted and converted into \texttt{json} format.
+As decribed in \autoref{sec:dataset}, documents are loaded as \texttt{xml} files from the database, and their ID, title, and body contents extracted and converted into \texttt{json} format. \autoref{prepro:example1} shows an example of an extracted document.
+\begin{figure}[h]
```

Comment:
Føler det er lidt underligt at have en quote som figure, men måske er det bare mig.

Reply:
Var lidt i tvivl om hvordan man bedst skulle lave et running example med text som kan blive refereret til.
Er åben for andre måder at gøre det på, dette var bare lige hvad jeg kunne tænke mig frem til